pkg/shell/index.html

```diff
@@ -365,7 +365,8 @@ <h4 class="modal-title" translatable="yes">Authentication</h4>
                     </div>
                 </div>
                 <div class="has-feedback search" id="menu-search">
-                    <input id="filter-menus" class="form-control" type="search" placeholder="Search" translate="placeholder">
+                    <input id="filter-menus" class="form-control" type="search" placeholder="Search"
+                        translate="placeholder" aria-label="Search" translate="aria-label">
```

Comment:
You cannot have two of the same attribute in HTML. (The translate attribute is here twice, which is invalid HTML.)

Reply:
hmm, but then we cannot translate the aria-label. I can use then hidden label I guess?

Reply:
Can't it support multiple values? For most other attributes, you just use space separation.

Example: `class="foo bar baz"`

If you use two attributes, then only the last will be set.

Reply:
(Oh, in some browsers, it's the first. In others, it's the last. It's undefined behavior, aside from it being not supported in HTML.)

Reply:
`translate="placeholder aria-label"` picks only `aria-label` for translation. We implement this functionality, so I guess we can fix it? I'll take a look

Reply:
ah no, scratch that, I mistyped and thats why it was not picked up. It works just fine. I pushed it here.